Makefile

```diff
@@ -54,7 +54,7 @@ endif
 
 ARCHFLAGS := $(ARCHFLAGS)
 
-OPTFLAGS  ?= -g -O
+OPTFLAGS  ?= -g -O3
```

Comment:
Typically in a collaborative software project, a feature branch should not change the compilation process. Such change should be made from the master branch. But I accept the change here.